application/src/main/java/org/thingsboard/server/service/entitiy/alarm/TbAlarmService.java

```diff
@@ -28,5 +29,9 @@ public interface TbAlarmService {
 
     ListenableFuture<Void> clear(Alarm alarm, User user);
 
+    ListenableFuture<Void> assign(Alarm alarm, User user, UserId assigneeId);
```

Comment:
No need for ListenableFuture. This operation goes to the PostgreSQL DB, and it is not async. Use similar pattern that we use in ```Alarm save(Alarm entity, User user)```